### PR TITLE
Update/wifi config corrections

### PR DESCRIPTION
**Pull request by shaunmulligan**

### CODE REVIEW

File: pages/configuration/custom-network.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 ### Multiple WiFi Networks
 
-We hope to eventually implement the ability to use Multiple WiFi SSIDs in the resin.io user interface however for the time being you can add this functionality manually by editing a configuration file on your SD card.
+We hope to eventually implement the ability to use Multiple WiFi SSIDs in the resin.io user interface however for the time being you can add this functionality manually by editing a configuration file on your `img`.
 
-In order to add additional networks edit`/resin/network/network.config` on the SD card either prior to the first execution of the device or on the `RECOVERY` partition after first execution.
+IOn the Raspberry Pi and Beaglebone, it is possible to change your wifi SSID or Passphrase after downloading the `.img`.
```
Comment (emirotin) [inline pages/configuration/custom-network.md]: IOn -> On

File: pages/configuration/custom-network.md [hunk 2]
```diff
@@ -4,9 +4,11 @@
 
 ### Multiple WiFi Networks
 
-We hope to eventually implement the ability to use Multiple WiFi SSIDs in the resin.io user interface however for the time being you can add this functionality manually by editing a configuration file on your SD card.
+We hope to eventually implement the ability to use Multiple WiFi SSIDs in the resin.io user interface however for the time being you can add this functionality manually by editing a configuration file on your `img`.
```
Comment (emirotin) [inline pages/configuration/custom-network.md]: maybe "on `img`" should be rephrased like "inside of the downloaded OS image"?

File: pages/configuration/custom-network.md [hunk 3]
```diff
@@ -4,9 +4,11 @@
...
 
-In order to add additional networks edit`/resin/network/network.config` on the SD card either prior to the first execution of the device or on the `RECOVERY` partition after first execution.
+IOn the Raspberry Pi and Beaglebone, it is possible to change your wifi SSID or Passphrase after downloading the `.img`. 
+
+Currently this can be done using the small config [writer tool](https://github.com/petrosagg/resin-net-config). This is a temporary tool and will be phased out very soon.
```
Comment (emirotin) [inline pages/configuration/custom-network.md]: Maybe just a link to the /using/ page here? Fewer places to change later

**Comment by craig-mulligan** [comment]
@shaunmulligan this looks fine to me, Okay to merge or we waiting on something?

**Pull request merged**